### PR TITLE
[cahnge] contents with better translation (KR)

### PR DESCRIPTION
**Pull request by ensia96**
Hi, there!
I'm web developer in korea! (o _o)/

I think that translation is not desirable.
So I changed the content that seemed to fit better.

### CODE REVIEW

**Comment by ensia96** [comment]
Now, I'm working on README file.

Plz do not merge this PR! ( _ _)

**Comment by ensia96** [comment]
I finished translating the README document.

I'll look around the storage more for the parts related to the configuration code, and then modify them with the new PR!

I owe you for your services. ( _ _)

Thank You!

File: docs/readme_kr.md [hunk 1]
```diff
@@ -10,7 +10,7 @@
     <a href="https://codecov.io/gh/anuraghazra/github-readme-stats">
       <img src="https://codecov.io/gh/anuraghazra/github-readme-stats/branch/master/graph/badge.svg" />
     </a>
-    <a href="https://github.com/anuraghazra/github-readme-stats/issues">
+    <a href="https://github.com/anuraghazra/github-readme-stas/issues">
```
Comment (Bas950) [inline docs/readme_kr.md:13]: Don't think you meant to change that.

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Comment by anuraghazra** [comment]
@ensia96 thanks for the PR!

File: docs/readme_kr.md (hunk 1 above)
Comment (ensia96) [reply docs/readme_kr.md]: Oops...

That's my type error <( o _o)

Thank you!

**Pull request merged**